src/Exceptions/HttpPreconditionFailed.php

```diff
@@ -8,4 +8,3 @@ class HttpPreconditionFailed extends HttpException {
     public const DESCRIPTION = "Indicates that one or more preconditions given in the request header fields evaluated to false when tested on the server.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/412";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces